github_api/prs.py

```diff
@@ -114,6 +114,17 @@ def label_pr(api, urn, pr_num, labels):
     return api("PUT", path, json=data)
 
 
+def close_stale_pr(api, urn, pr, delta, reason):
```

Comment:
Maybe a better name like `handle_broken_pr`? This implies the PR is stale and it will close it.